### PR TITLE
Biotools new

### PR DESCRIPTION
**Pull request by osallou**
put biocontainers tools with no biotools entry found

### CODE REVIEW

File: data/Aliscore/biocontainers.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "softwares": {
+        "Aliscore": {
```
Comment (bgruening) [inline data/Aliscore/biocontainers.json]: @osallou can you change the folder name to be lower-case ...

File: data/BIOR/biocontainers.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (bgruening) [inline data/BIOR/biocontainers.json]: lower case would be great

**Comment by osallou** [comment]
I have added as biocontainers.json only those from our github repo.
For containers from bioconda i think that only conda.json files shoild be ok, as they all have a related container

**Comment by bgruening** [comment]
> For containers from bioconda i think that only conda.json files shoild be ok, as they all have a related container

Yes. But I guess @ypriverol wants to have more metadata in those files.

Can you change the filenames?

**Pull request closed without merging**